lib/spack/spack/test/conftest.py

```diff
@@ -159,7 +159,9 @@ def latest_commit():
             return git("rev-list", "-n1", "HEAD", output=str, error=str).strip()
 
         # Add two commits on main branch
-        write_file(filename, "[]")
+
+        # A commit without a previous version counts as "0"
+        write_file(filename, "[0]")
```

Comment:
ah, okay. weird.

Reply:
See https://github.com/spack/spack/pull/35681#discussion_r1122232365 The git version was satisfying `@:0` so I tried to maintain the same high-level semantic, though I think trying to map commit hash to version ranges is not doable in general.